src/main/java/htsjdk/variant/vcf/VCFHeader.java

```diff
@@ -604,6 +604,10 @@ private VCFHeaderVersion initializeHeaderVersion() {
         return metaDataVersion;
     }
 
+    public Collection<VCFValidationFailure<VCFHeaderLine>> getValidationErrors(final VCFHeaderVersion targetVersion) {
+        return mMetaData.getValidationErrors(targetVersion);
+    }
```

Comment:
If the header upgrade code gets moved right into `VCFHeader` (see my comments elsewhere), then this doesn't have to be exposed at all. But if we keep it, it should get javadoc, and be moved in with the other public methods.